docs/source/rest_api/error_codes.rst

```diff
@@ -46,12 +46,12 @@ include a single *error* property with three values:
 
    * *code* (integer) - a machine readable error code
    * *title* (string) - a short headline for the error
-   * *message* (string) - a longer human readable description of what went wrong
+   * *message* (string) - a longer, human-readable description of what went wrong
 
 .. note::
 
-   While the title and message may change in the future, **the error code is
-   fixed**, and will always refer to this particular problem.
+   While the title and message may change in the future, the error code
+   will **not** change; it will always refer to this particular problem.
```

Comment:
COMMENT FROM ORIG PR #1222 : 
_delventhalz 6 hours ago  Member_
For what it's worth, the term "fixed" means more to me than "will not change". The first feels like a contractual definition, while the second is a vague aspiration. Perhaps the clause after the semicolon could become: ; it is fixed and will always refer...